Firebase/InAppMessagingDisplay/FIDRenderingWindowHelper.m

```diff
@@ -24,7 +24,16 @@ + (UIWindow *)UIWindowForModalView {
   static dispatch_once_t onceToken;
 
   dispatch_once(&onceToken, ^{
-    UIWindowForModal = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
+    if (@available(iOS 13.0, *)) {
+      UIWindowScene *foregroundedScene = [[self class] foregroundedScene];
```

Comment:
Could this ever be `nil` on iOS 13?

Reply:
Yes, could be `nil` if this is called there are no foregrounded scenes. In that case it should be a no-op. Though, now that you mention it I should force the nil case to make sure nothing weird happens.

Reply:
Tested this, it's a no-op as expected.